src/Foundatio.Redis/Scripts/DequeueId.lua

```diff
@@ -1,7 +1,7 @@
 ﻿local item = redis.call('RPOPLPUSH', @queueListName, @workListName);
 if item then
-    local dequeuedTimeKey = 'q:' .. @queueName .. ':' .. item .. ':dequeued';
-    local renewedTimeKey = 'q:' .. @queueName .. ':' .. item .. ':renewed';
+    local dequeuedTimeKey = @listPrefix .. ':' .. item .. ':dequeued';
```

Comment:
Where is @listPrefix defined?

Reply:
Ahh nevermind, it's in the call. Was thinking it had to be defined in the script.